src/Symfony/Component/ErrorHandler/DebugClassLoader.php

```diff
@@ -848,6 +848,10 @@ private function setReturnType(string $types, string $class, string $method, str
         $phpTypes = [];
         $docTypes = [];
 
+        if (\count($typesMap) > 1 && array_key_exists('callable', $typesMap)) {
```

Comment:
wouldn't this also return for _union_ types here as you don't check `$glue` ?

Reply:
Mhh my bad, I made a mistake where it's not possible do declare a property type using `callable` but it's actually allowed to use `union` between callable and other types. Thanks!